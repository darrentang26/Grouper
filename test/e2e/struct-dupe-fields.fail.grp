type Coord = {x : Int, x : Int}

let Coord origin = {x = 0, x = 0} in
print "made origin"