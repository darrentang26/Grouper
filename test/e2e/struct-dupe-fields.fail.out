Fatal error: exception Semant.Failure("Struct field names must be unique")
Undefined symbols for architecture arm64:
  "_main", referenced from:
     implicit entry/start for main executable
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)